mogc_partnerships/views.py

```diff
@@ -192,6 +192,77 @@ def get_serializer_context(self):
         return {"cohort": cohort}
 
 
+class CohortMembershipUpdateView(generics.UpdateAPIView):
```

Comment:
Does this prevent non-managers from updating a membership?

Reply:
Ah, no, it does not. We're currently doing that by filtering on the user's partners and raising a permission denied error if none are found, correct?

Reply:
Okay, I pushed an update with an initial stab at create custom permissions classes for managers working with cohorts and partners, which I'm going to work on refining.

Reply:
Okay, I'm ready to merge this if you are 🤜🤛 Let me know if there is any refinement I should wait on.

Reply:
Thanks! I think we're good for now, I created issues for the two improvements we can hopefully tackle later on.